source/solvers/heat_transfer_assemblers.cc

```diff
@@ -92,7 +92,7 @@ HeatTransferAssemblerCore<dim>::assemble_matrix(
 
       // Calculate the artificial viscosity of the shock capture as in equation
       // (79) of Tezduyar 2003
-      const double nu_dcdd = 0.5 * h * h * u_mag *
+      const double nu_dcdd = is_steady(method) ? 0 : 0.5 * h * h * u_mag *
                              dcdd_temperature_gradient.norm() /
```

Comment:
So you remove DCDD for steady simulations?

Reply:
Yep! It is too non-linear without this.